tests/installation/releasenotes.pm

```diff
@@ -23,7 +23,8 @@ sub run() {
     }
 
     # workaround for bsc#1014178
-    if (check_screen('zfcp-popup'), 0) {
+    wait_still_screen(5);
+    if (check_screen('zfcp-popup')) {
```

Comment:
http://opeth.suse.de/tests/4400#step/releasenotes/4

`15:31:03.0220 11830 <<< testapi::check_screen(mustmatch='zfcp-popup', timeout=30)
15:31:03.0826 11831 MATCH(releasenotes-zfcp-popup-20161207:1.00)
15:31:03.1490 11830 >>> testapi::_check_backend_response: found releasenotes-zfcp-popup-20161207, similarity 1.00 @ 382/373
15:31:03.1495 Debug: /var/lib/openqa/share/tests/sle/tests/installation/releasenotes.pm:36 called testapi::send_key
15:31:03.1496 11830 <<< testapi::send_key(key='alt-e')`

it continues directly with  alt-e